tests/cases/fourslash/referencesForStringLiteralPropertyNames7.ts

```diff
@@ -5,7 +5,7 @@
 // @checkJs: true
 
 ////var x = { [|"[|{| "isWriteAccess": true, "isDefinition": true, "contextRangeIndex": 0 |}someProperty|]": 0|] }
-////x["[|someProperty|]"] = 3;
+////x["[|{| "isWriteAccess": true, "isDefinition": true |}someProperty|]"] = 3;
 ////[|x.[|{| "isWriteAccess": true, "isDefinition": true, "contextRangeIndex": 3 |}someProperty|] = 5;|]
```

Comment:
Similar question here: why these two lines are marked as definition if actual property defined in object, and this two are just writes to that property? Also in TS are just writes too.
Am I missing something?